src/main/java/com/powsybl/openloadflow/network/impl/LfControlAreaImpl.java

```diff
@@ -0,0 +1,75 @@
+package com.powsybl.openloadflow.network.impl;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.openloadflow.network.*;
+import com.powsybl.openloadflow.util.Evaluable;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Supplier;
+
+public class LfControlAreaImpl extends AbstractPropertyBag implements LfControlArea {
+
+    private final LfNetwork network;
+    private final Ref<Area> areaRef;
+
+    private final double targetAcInterchange;
```

Comment:
Rename it to interchangeTarget

Reply:
changed in 63eb39786f9f0e1cfa606e8f9471591b54a34da5